service-workers/service-worker/fetch-event-referrer-policy.https.html

```diff
@@ -7,12 +7,22 @@
 <script>
 var worker = 'resources/fetch-event-test-worker.js';
 
+if (window.internals && window.internals.settings)
+    internals.settings.setAllowDisplayOfInsecureContent(true);
```

Comment:
@youennf What is this?  Generally this sort of thing is managed in the harness integration with the browser vendor's test suite.  We don't usually put non-standard stuff like this in the WPT to my knowledge.

Reply:
JFYI for Blink we're somehow able to use testharnessreport.js to add internal settings. Can WebKit have a similar setup? For reference here's the change to the file in the patch foolip linked to above:
https://chromium-review.googlesource.com/c/chromium/src/+/567963/3/third_party/WebKit/LayoutTests/resources/testharnessreport.js

Reply:
Thanks @matto.
We are also using testharnessreport.js for these kind of setup but not on a per test basis right now.
I guess this might be fine as a temporary solution.
Adding this kind of configuration within the test itself using something like window.test_driver might be a better approach as other browser teams will more easily understand what needs to be done.